expressServer/scripts/encryptExistingUsers.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+const dotenv = require('dotenv');
+const app = require('express');
+const crypto = require('crypto');
+
+dotenv.config();
+
+mongoose.connect(process.env.MONGODB_URI, { useNewUrlParser: true, useUnifiedTopology: true })
+  .catch(err => console.error('Could not connect to MongoDB', err));
+
+const userSchema = new mongoose.Schema({
+    username: { type: String, required: true, unique: true },
+    hash: { type: String, required: true },
+    salt: { type: String, required: false }
+});
+
+const User = mongoose.model('User', userSchema);
+
+function hashPassword(password, salt) {
+    return crypto.createHash('sha3-512').update(password + salt).digest('hex');
+}
+
+async function encryptAllUserPasswords() {
+    try {
+        const users = await User.find({});
+        for (let user of users) {
```

Comment:
How will you handle this in memory if you have 10.000.000 of users? do you think this is a good approach or do you prefer to do save using batches?